Patch by Zarafa <http://www.zarafa.com/> - when a line in a plain text mail starts with a '.', the
character needs to be escaped. VMime has a special filter for this, but due to a bug in this filter, a
second line starting with a '.' would trigger a wrong buffer copy, and your email would contain double
parts.

--- libvmime-0.7.1/src/utility/filteredStream.cpp			2005-06-16 21:50:12.000000000 +0200
+++ libvmime-0.7.1/src/utility/filteredStream.cpp.out-of-bounds-copy	2007-06-15 14:29:40.907779750 +0200
@@ -146,7 +146,7 @@
 
 		if (previousChar == '\n')
 		{
-			m_stream.write(start, pos - data);
+			m_stream.write(start, pos - start);
 			m_stream.write("..", 2);
 
 			start = pos + 1;
